### DISCUSSION TITLE
Swiching power LED state when mic is turned on\off or enchance muted\unmuted popup.

### DISCUSSION THREAD

**Discussion by lHolyDiverl** [seerge/g-helper#3481] [Ideas]
Hello,

I own ROG Strix G713RM laptop which has a huge red LED located at the top of power button.

It has only one function which is to be turned on when laptop is turned on too which is pretty obvious.

I would like to make it more practical, e.g. by:
 - Turn it off when mic is muted.
 - Turn it on when mic is unmuted.
 
You can forget in which state currently is mic when e.g.  on meetings you need to periodically mute\unmute your mic.

This approach would make this LED more practical and assign some sort of functionality to it :) 

Alternative solution:
Make an option to have Muted\Unmuted popup to displayed whole time + with option to changed it position.
And some checkbox\hotkey to toggle if current or 'permanent popup' will be displayed.

Both cases will provide full visual control of current mic state.

Thank you.

P.S. 2 years ago I wrote to ASUS that it would be nice if they just add a single LED for M3 MIC button (like capslock has) to represent current mic state which option to enabled it if needed. As always, they reply that they will transfer this idea to developer teams and looks like nothing change since that time in new models.

**Comment by seerge** [comment]
@lHolyDiverl hello, 

Those LEDs can't be controlled so unfortunately you can't neither toggle them on or off. 

Thanks.
